Mitgliedsantrag_ZTL.tex

```diff
@@ -111,13 +129,12 @@
 
```

Comment:
Was ist der Sinn von "beider" Erziehungsberechtigten

Reply:
Damit erreichen wir, dass beide Erziehungsberechtigten (aka Elternteile) ihr Einverständnis geben. Tun wir das nicht, kann es zu unschönen Diskussionen bei getrennt lebenden Erziehungsberechtigten kommen. In meinen anderen Vereinen legen wir ebenfalls Wert auf die Unterschrift beider Erziehungsberechtigten, um  Meinungsverschiedenheiten mit dem getrennt lebenden Elternteil nicht auf dem Rücken des Vereins austragen zu müssen. Hat sich bewährt, wollen wir hier auch so machen.